backend/tracim_backend/applications/share/__init__.py

```diff
@@ -0,0 +1,4 @@
+from tracim_backend.applications.share.application import get_app  # noqa: F401
+from tracim_backend.applications.share.config import check_config  # noqa: F401
+from tracim_backend.applications.share.config import load_config  # noqa: F401
+from tracim_backend.applications.share.controller import import_controller  # noqa: F401
```

Comment:
See with @lebouquetin about use `__ini__`files. Import proposition to avoid them can be:

``` python
import tracim_backend.applications.upload_permissions.config as upload_permissions_config

upload_permissions_config.load_config
```

Reply:
changed.